jsapp/scss/components/_kobo.asset-row.scss

```diff
@@ -163,9 +167,17 @@ $hover-asset-row: #F2F2F3;
   &:hover {
     > .asset-row__buttons {
       opacity: 1;
-      width: 33%;
-      overflow: visible;
-      transition: 0.3s opacity;
+      visibility: visible;
+      transition: $hover-transition;
+      // make them appear moment after items behind starts to disappear
+      transition-delay: 0.1s;
+    }
+
+    // hide all columns but first one
+    .asset-row__cell--asset-details > .asset-row__cell:not(:first-of-type) {
+      opacity: 0;
+      visibility: hidden;
+      transition: $hover-transition;
```

Comment:
This is not the desired behaviour. In the previous commit, I specifically removed the width of `.asset-row__buttons` so that we could see the `Created` or `Modified` value of that row even on hover.

Reply:
Oh, ok; removed it :) (modified is actually being hidden by some code that was already there)